snake.go

```diff
@@ -216,6 +216,27 @@ func (g *Game) moveSnake() {
 	g.snake.body[0][1] = y
 }
 
+func (game *Game) drawApple(x, y int16) {
+	// Those variable are there for a more easy reading of the apple shape
+	r := game.colors[APPLE]
+	b := game.colors[BCK]
+	g := game.colors[SNAKE]
+	// The array is split for a visual purpose too
+	applebuffer := []color.RGBA{
```

Comment:
This should be `const` or moved outside the func to the top, to minimize memory allocations

Reply:
As we can not define an array as constant, is it okay to move these variables to the top of the file?